File: Right-angle-triangle/Right-angle.py
angleA = int(input("Enter First angle: "))
angleB = int(input("Enter Second angle: "))

angleC = 180 - (angleA + angleB)

if angleA == 90 or angleB == 90 or angleC == 90:
    print("Right-angled Triangle")
else:
    print("Not a right-angled triangle")

File: Right-angle-triangle/Right-angle.c
#include <stdio.h>

int main() {
    int angleA, angleB, angleC;

    printf("Enter first angle: ");
    scanf("%d", &angleA);

    printf("Enter second angle: ");
    scanf("%d", &angleB);

    angleC = 180 - (angleA + angleB);

    if (angleA == 90 || angleB == 90 || angleC == 90) {
        printf("Right-angled triangle\n");
    } else {
        printf("This is not a right-angled triangle\n");
    }

    return 0;
}
